aya/src/maps/mod.rs

```diff
@@ -166,6 +205,29 @@ impl Map {
             name: self.obj.name.clone(),
         })
     }
+
+    pub fn pin<P: AsRef<Path>>(&mut self, path: P) -> Result<(), MapError> {
```

Comment:
This needs docs and an example. (I know the other map methods don't have docs, but this is the first method you'd actually use from the generic `Map` type.)

Reply:
I've made this `pub(crate)` for now too since it's only ever called from within a `load` or `load_file`.
As such, I haven't added any docs as it's not a public API, but I can do if you'd like me to.